g11n-ws/vip-manager-i18n/src/main/java/com/vmware/vip/core/conf/CustomErrorController.java

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright 2019-2020 VMware, Inc.
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package com.vmware.vip.core.conf;
+import org.springframework.boot.web.servlet.error.ErrorController;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+public class CustomErrorController implements ErrorController {
+
+    private static final String PATH = "/error";
+
+    @RequestMapping(value = PATH)
```

Comment:
Suggest to change name 'PATH' to ErrorURL or others. It's confused to see it first time, hard to understand.

Reply:
It's not URL, it's a default path to map error static resource.